OrbitQt/orbittreeview.cpp

```diff
@@ -110,7 +110,9 @@ void OrbitTreeView::OnSort(int section, Qt::SortOrder order) {
   }
 
   model_->sort(section, order);
+  model_->GetDataView()->SetIsFilteringOrSorting(true);
   Refresh();
+  model_->GetDataView()->SetIsFilteringOrSorting(false);
```

Comment:
Is there any way we could remove the state setting here? Consider passing a "RefreshMode" enum as argument to Refresh instead maybe?

Reply:
Good idea! Changed to use the `enum class RefreshMode`. Thanks so much for the suggestion!